cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconcilerStatusTest.java

```diff
@@ -181,6 +184,87 @@ public void testKafkaReconcilerStatus(VertxTestContext context) {
         }));
     }
 
+    @Test
+    public void testKafkaReconcilerStatusUpdateVersion(VertxTestContext context) {
+        Kafka kafka = new KafkaBuilder(KAFKA)
+                .editOrNewSpec()
+                .editOrNewKafka()
+                .withReplicas(1)
+                .endKafka()
+                .endSpec()
+                .editOrNewStatus()
+                // supportedVersions is an ordered set, get the first element for the lowest kafka version
+                .withKafkaVersion(VERSIONS.supportedVersions().iterator().next())
```

Comment:
Would it be more readable if you use something like `KafkaVersionTestUtils.PREVIOUS_KAFKA_VERSION`?